app/models/concerns/resources.rb

```diff
@@ -0,0 +1,8 @@
+module Resources
+  VALID_PASSWORD_REGEX =
+    /\A^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[$&+,:;=?@#|'<>.-^*()%!])[A-Za-z\d$&+,:;=?@#|'<>.-^*()%!]{8,}$\z/
+  VALID_NAME_REGEX = /\A[A-Za-z]+\z/
+  VALID_LOCATION_REGEX = /\A[A-Za-z\s]+\z/
```

Comment:
do we need to consider spaces in Location name? eg: `Dh aka` will be act as valid here right?

Reply:
Vaia, locations may have names with multiple words like, New York or Old Dhaka